Felienne/hedy
# [UI IMPROVEMENT] Starts turtle at bottom of screen going up

**Description**
In this PR we change the starting position and direction of the turtle when making a drawing program. Currently the turtle starts (ommenabij) in the middle of the screen facing to the right. This is changed to started at the middle-bottom of the screen facing up, like car in a game.

**Edit**
To better work with #2826 we decided to start in the middle of the screen facing up instead of at the bottom facing up. So effectively the only change is the arrow pointing another direction at the start of the program.

**Fixes**
This PR fixes #2730.

**How to test**
Navigate to any Hedy level and use the Turtle. Verify that is always start at the middle-bottom of the screen.
